amivapi/utils.py

```diff
@@ -74,15 +74,21 @@ def get_id(item):
         return ObjectId(item['_id'])
 
 
-def mail(sender, to, subject, text):
+def mail(to, subject, text):
     """Send a mail to a list of recipients.
 
+    The mail is sent from the address specified by `API_MAIL` in the config,
+    and the subject is prefixed by `API_MAIL_SUBJECT_PREFIX`.
+
+
     Args:
-        from(string): From address
         to(list of strings): List of recipient addresses
         subject(string): Subject string
         text(string): Mail content
     """
+    sender = app.config['API_MAIL']
+    subject = " ".join((app.config['API_MAIL_SUBJECT_PREFIX'], subject))
```

Comment:
Instead of a configurable prefix, I think a template string is more intuitive when looking into the config.

Reply:
Fair enough